### PR TITLE
Various fixes

### LINKED ISSUE #38: Access modes and presentational content
We don't explicitly state whether to set access modes for content that is purely presentational.

If a book has a few decorative images, for example, does it still have a visual access mode? Does it have both a visual and textual sufficient access mode and a textual one, or does it only have a textual access mode?

I've always assumed that you wouldn't set an access mode for presentational content, but I'm going solely by the word "information" in the definition.

Would help to be clearer on this.

### PR DESCRIPTION
**Pull request by mattgarrish**
My intention when I started was only to fix issue 38, but I got pulled into doing a number of other fixes. To recap, this pull request:

- clarifies that access modes do not apply to presentational content. I've added notes to each of the three main categories (auditory, textual and visual) to additionally explain when they don't apply. I also put the text about sufficient access modes in the access modes definition into a note as it seemed a bit out of place as body text.
- clarifies that single-value sufficient access modes are the most important to list (to match what we're doing with the epub techniques).
- identifies the expected values for properties and recommends how to handle for less expressive formats (i.e., formalize our use of comma-separated lists).
- identifies hazard and feature property values that must not be specified together to avoid ambiguity (e.g., not to specify none when a hazard is also specified). While they should be intuitive, I don't think we verify these in ace because we've never mentioned that this should be done.

Fixes #38

### CODE REVIEW

**Comment by madeleinerothberg** [comment]
> The access modes do  not tell users if all the specified modes are necessary to consume the information or if only individual modes or combinations are necessary (e.g., in a book with audio content, the ability to read textual content may only be necessary if transcripts are provided).

I'm confused by this example. Do you mean the ability to read textual content may not be necessary if the only text provided is the transcript? Meaning audio is sufficient?

**Comment by mattgarrish** [comment]
> (e.g., in a book with audio content, the ability to read textual content may only be necessary if transcripts are provided)

Ya, this does sound weird. I dropped it into the note without re-reading it, but "may only be necessary" looks like the wrong wording here. The expectation is that textual is sufficient if transcripts are provided, not that it's optional without them, so I would rephrase it as:

> (e.g., in a book with audio content, the ability to read textual content may be sufficient if transcripts are provided)

**Comment by GeorgeKerscher** [comment]
This revised text sounds good.
